.github/workflows/tardis-research-papers.yml

```diff
@@ -0,0 +1,117 @@
+name: tardis-research-papers
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  NASA_ADS_TOKEN: ${{ secrets.NASA_ADS_TOKEN }}
+
+jobs:
+    research-using-tardis:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Setup Python
+              uses: actions/setup-python@v5
+              with:
+                python-version: "3.x"
+    
+            - name: Install dependencies
+              run: |
+                python -m pip install --upgrade pip
+                pip install jupyter nbconvert
+                        
```

Comment:
We're using micromamba action, so just for consistency can you try using that?
https://github.com/mamba-org/setup-micromamba#create-environment-from-environment-specs